content/posts/2015-03-02_march-newsletter.md

```diff
@@ -0,0 +1,103 @@
+Title: March Newsletter
+Date: 2015-03-02
+Tags: newsletter
+Slug: march-newsletter-2015
+Author: David Fischer
+
+
+Pythonistas,
+
+If there are articles, jobs or events you'd like to see in the next newsletter
+or if you have any feedback on San Diego Python, feel free to
+[share with us][].
+
+[share with us]: mailto:sandiegopython-organizers@googlegroups.com
+
+
+Upcoming events
+---------------
+
+[Saturday study group][saturday-meetup] (meetup.com) <br />
+San Diego PyLadies and SD Python are continuing the Saturday study group
+at the **Ansir Center**. The goal of the Saturday study group is to learn Python
+together in a collaborative environment.
+
+[saturday-meetup]: http://www.meetup.com/pythonsd/events/220475825/
+
+
+[Monthly Meetup][] (meetup.com) <br />
+Our regular monthly meetup will be held at the Ansir Innovation Center on
+Thursday March 26 at 7pm. We are doing 5 minute lightning talks. If you
+are interested in speaking, please [contact us][] or post in the meetup
+comments.
+
+[Monthly Meetup]: http://www.meetup.com/pythonsd/events/220848308/
+[contact us]: mailto:sandiegopython-organizers@googlegroups.com
+
+
+Jobs
+----
+
+[AWS systems engineer][bs-engineer] (brightscope.com) <br />
+BrightScope, a local financial information company in UTC, is looking for
+an exceptional engineer to help with their AWS infrastructure. They are big
+San Diego Python supporters.
+
+[bs-engineer]: http://www.brightscope.com/about/careers/#job_AWS_Systems_Engineer
```

Comment:
Instead of the brightscope link can we use
https://careers.stackoverflow.com/jobs/81143/aws-system-engineer-brightscope-inc